odk1-src/form-styling.rst

```diff
@@ -27,23 +27,57 @@ Media
 
 You can include questions in your form that display images or that play video or audio files by including a :th:`media` column in your `XLSForm <http://xlsform.org/#media>`_.
 
-.. image:: /img/form-styling/media-image.* 
+Images
+~~~~~~~~
+
+Adding image to a question adds a displays the image as part of the question.
```

Comment:
"Adding image to a question adds a displays" -> "Adding image to a question displays"

Reply:
Good catch!